guides/common/modules/ref_virt-who-configuration-examples.adoc

```diff
@@ -0,0 +1,98 @@
+[id="virt-who-configuration-examples_{context}"]
+= Virt-who configuration examples
+
+You configure virt-who in a file that specifies details such as the virtualization type and the hypervisor or virtualization manager to query.
+
+[IMPORTANT]
+====
+The `rhevm` and `xen` hypervisors are not supported.
+
+The `kubevirt` hypervisor is a Technology Preview.
+====
+
+.Generic example
+
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+type=libvirt <1>
+hypervisor_id=hostname
+owner=Default_Organization
+env=Library
+server=_hypervisor.example.com_
+username=_virt_who_user_ <2>
+encrypted_password=$cr_password
+rhsm_hostname=_{foreman-example-com}_
+rhsm_username=virt_who_reporter_1
+rhsm_encrypted_password=$user_password
+rhsm_prefix=/rhsm
+----
+<1> Hypervisor type.
+<2> Read-only user that you create on the hypervisor before configuring virt-who.
+
+.{OpenStack} example
+
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+type=libvirt
+hypervisor_id=hostname
+owner=ORG
+env=Library
+server=qemu:///system   <====
```

Comment:
Is this an open TODO?

Reply:
Who knows? It has been there for years. Honestly, we can probably live without a generic OpenStack virt-who config example. :smile:

Reply:
No idea. It's been there for years. I created a bug to fix this module, so we can resolve this thread.